spring-cloud-gcp-samples/spring-cloud-gcp-integration-pubsub-sample/spring-cloud-gcp-integration-pubsub-sample-sender/src/main/resources/static/index.html

```diff
@@ -1,6 +1,6 @@
 <!DOCTYPE html>
 <!--
-  ~  Copyright 2017 original author or authors.
+  ~  Copyright 2017-2018 the original author or authors.
```

Comment:
No Copyright in the non-Java(Kotlin, Groovy) files.
In other words: no Copyright, please, in the XML and HTML files.